CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ endif()
 foreach(p
   CMP0048 # CMake 3.0
   CMP0054 # CMake 3.1
+  CMP0020 # CMake 3.0.2
```

Comment:
I believe these versions are when the policies were introduced, so it should be 2.8.11 for CMP0020.